README.md

```diff
@@ -2,11 +2,11 @@
 Installs [kafka](https://kafka.apache.org/)
 
 ##Requirements
+- kafka_hosts - comma separated list of host:port pairs in the cluster, defaults to 'ansible_fqdn:9092' for a single node 
```

Comment:
I had left it in the requirements section because you must change it for a cluster, I noted in that line the default is only for a single node.